Pipfile

```diff
@@ -5,6 +5,8 @@ verify_ssl = true
 
 [dev-packages]
 flake8 = "*"
+pytest = "*"
```

Comment:
this is not needed
once you have django pytest, 
you can run it locally to test it